internal/authn/oidc/oidc.go

```diff
@@ -148,7 +153,7 @@ func (oidc *RemoteOidcAuthenticator) GetKeys() (*keyfunc.JWKS, error) {
 }
 
 func (oidc *RemoteOidcAuthenticator) GetConfiguration() (*authn.OidcConfig, error) {
-	wellKnown := strings.TrimSuffix(oidc.IssuerURL, "/") + "/.well-known/openid-configuration"
+	wellKnown := strings.TrimSuffix(oidc.IssuerURLs[0], "/") + "/.well-known/openid-configuration"
```

Comment:
```suggestion
	wellKnown := strings.TrimSuffix(oidc.MainIssuer, "/") + "/.well-known/openid-configuration"
```

to make it more obvious